values.yaml

```diff
@@ -320,10 +320,11 @@ hooks:
           STOREPASS="{{- .Values.ssl.keystorePassword -}}"
           KEYPASS="{{- .Values.ssl.keystoreKeypassword -}}"
           KEYTYPE="{{- .Values.ssl.keystoreType -}}"
-          keytool -genkey -keyalg RSA -alias dai-deploy -keystore conf/keystore.$KEYTYPE -validity 365 -keysize 2048 -storepass "$STOREPASS" -storetype "$KEYTYPE" -keypass "$KEYPASS" \
+          STORETYPE="{{- if eq .Values.ssl.keystoreType "p12" -}}pkcs12{{- else if eq .Values.ssl.keystoreType "jks" -}}jks{{- end -}}"
```

Comment:
Why are we renaming store type from `pkcs12` to `p12`, and because of that we need here logic to put it back to `pkcs12` value?

Reply:
existing .Values.ssl.keystoretype is used interchangeably for both the file extension and storetype. which does not work. Here for generation, storetype requires pkcs12 value for keytool command.